kodi_addon_checker/check_addon.py

```diff
@@ -458,18 +458,31 @@ def _check_for_existing_addon(report: Report, addon_path, all_repo_addons):
         repo_addons = all_repo_addons[branch]
 
         if addon_name in repo_addons:
-            if LooseVersion(addon_version) <= LooseVersion(repo_addons[addon_name]):
-                report.add(Record(PROBLEM, "%s addon already exist with version: %s in %s branch"
-                                  % (addon_name, repo_addons[addon_name], branch)))
-                return
-            else:
-                report.add(Record(INFORMATION, "%s addon also exists in %s branch but with version: %s"
-                                  % (addon_name, branch, repo_addons[addon_name])))
-                return
+            _check_versions(report, addon_name, branch, addon_version, repo_addons[addon_name], pr)
+            return
 
     report.add(Record(INFORMATION, "This is a new addon"))
 
 
+def _check_versions(report: Report, addon_name, branch, addon_version, repo_addons_version, pr):
+    if pr:
+        if LooseVersion(addon_version) > LooseVersion(repo_addons_version):
+            LOGGER.info("Addon version is %s " % addon_version)
+            return
```

Comment:
`return` not needed, no further code of that function will be executed without it